### PR TITLE
Run dns performance tests with node-cache enabled.

### PR DESCRIPTION
**Pull request by prameshj**

### CODE REVIEW

File: config/jobs/kubernetes/sig-network/sig-network-misc.yaml [hunk 1]
```diff
@@ -118,6 +118,82 @@ periodics:
...
+      - --extract=ci/latest
+      - --gcp-master-image=gci
+      - --gcp-node-image=gci
+      - --gcp-nodes=3
+      - --gcp-zone=us-east1-b
+      - --provider=gce
+      - --test_args=--ginkgo.focus=\[Feature:PerformanceDNS\]
+      - --timeout=60m
+      - --use-logexporter
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20181105-66d3bcb98-master
```
Comment (MrHohn) [inline config/jobs/kubernetes/sig-network/sig-network-misc.yaml]: Here an below, might need to update to the latest version:
```
gcr.io/k8s-testimages/kubekins-e2e:v20181129-33929460d-master
```

File: config/jobs/kubernetes/sig-network/sig-network-misc.yaml [hunk 2]
```diff
@@ -264,6 +364,31 @@ periodics:
       - --timeout=500m
       image: gcr.io/k8s-testimages/kubekins-e2e:v20181105-66d3bcb98-master
 
+- interval: 30m
+  name: ci-kubernetes-e2e-gci-gce-serial-kube-dns-nodecache
```
Comment (MrHohn) [inline config/jobs/kubernetes/sig-network/sig-network-misc.yaml]: Do we need `ci-kubernetes-e2e-gci-gce-serial-coredns-nodecache`?

Comment (prameshj) [reply config/jobs/kubernetes/sig-network/sig-network-misc.yaml]: There is no gci-gce-serial-coredns

Comment (MrHohn) [reply config/jobs/kubernetes/sig-network/sig-network-misc.yaml]: Ah got it, thanks!

**Pull request merged**